docs/api/apiv3/components/schemas/work_package_model.yml

```diff
@@ -143,11 +143,14 @@ properties:
             **Permission**: add watcher
           readOnly: true
       customActions:
-        allOf:
-        - "$ref": "./link.yml"
-        - description: Collection of predefined changes that can be applied to the
-            work package
-          readOnly: true
+        type: array
+        readOnly: true
+        items:
+          allOf:
+          - "$ref": "./link.yml"
+          - description: Collection of predefined changes that can be applied to the
```

Comment:
the description no longer matches. It should instead be s.th. like

A predefined action that can be applied to the work package.

**Resource**: CustomAction